traveler-swift-core/TravelerKit/AuthPath.swift

```diff
@@ -132,7 +132,7 @@ enum AuthPath {
         case .cancelOrder(let request):
             urlComponents.path = "/v1/order/\(request.quote.order.id)/cancellation/\(request.quote.id)"
             urlRequest.jsonBody = [
-                "cancellationReason": request.reason,
+                "cancellationReason": request.reason.id,
```

Comment:
This does not belong in this PR. UI PRs should only contain UI layer changes. This is a core change.